src/components/ha-date-range-picker.ts

```diff
@@ -76,7 +83,10 @@ export class HaDateRangePicker extends LitElement {
           ></paper-input>
         </div>
         ${this.ranges
-          ? html`<div slot="ranges" class="date-range-ranges">
+          ? html`<div
+              slot="ranges"
+              class=${classMap({ "date-range-ranges": true, rtl: this._rtl })}
```

Comment:
```suggestion
              class="date-range-ranges ${classMap({rtl: this._rtl })}"
```

Reply:
Instead of setting a class, why not just set the direction (`dir`) attribute?

Reply:
OK. Fixed. So priority is local attribute if applicable, classmap, custom CSS attribute.